### ISSUE TITLE
Include `touch` option on `save`

### ISSUE THREAD

**Issue by rubenrails** [amco/dolly#88]
@netlogics recommended adding a touch option when saving a record. Example offered was:

```ruby
if timestamps
  set_created_at
  set_updated_at unless options[:touch] == false
```

**Comment by javierg** [comment]
I don't like too many options on save... We want to have a skinny api

**Comment by netlogics** [comment]
Validate and touch are the only two I'm aware of. 
I would propose a definition for skinny API to mean
`Exclusive feature set with comprehensive implementation`
By carefully adhering to a proposed standard we can maintain a skinny API with consistent and expected functionality. Skinny API doesn't mean less lines of code. It means defining and sticking to a core feature set. Conversely comprehensive implementation doesn't mean more lines of code. It means implementing the core feature set so it is complete and functions in a manner that is consistent with ruby's and rails' conventions.

**Comment by javierg** [comment]
Exactly... The implenenyation of timestamps IMO should be left to the App... So, leaving the handling of timestamps should be ob the timestamps method not con save

**Comment by netlogics** [comment]
I don't think it's necessary to strictly parallel rails implementation. Validate was essential for consistency imo but touch probably isn't.
